### PR TITLE
Make version of fabric-gateway controller configurable

### PR DESCRIPTION
**Pull request by RomanZavodskikh**
Such change is needed to deploy fix to the fabric-gateway in a gradual way.

### CODE REVIEW

**Comment by RomanZavodskikh** [comment]
:+1:

File: cluster/manifests/fabric-gateway/deployment.yaml [hunk 1]
```diff
@@ -8,7 +8,11 @@ metadata:
   labels:
     application: gateway-operator
     component: fabric-gateway-controller
+  {{ if .ConfigItems.fabric_gateway_controller_version }}
+    version: "{{ .ConfigItems.fabric_gateway_controller_version }}"
+  {{ else }}
     version: "{{ $version }}"
+  {{ end }}
```
Comment (AlexanderYastrebov) [inline cluster/manifests/fabric-gateway/deployment.yaml]: Lets add fabric_gateway_controller_version into config-defaults with current value and remove $version form the top of the file

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by RomanZavodskikh** [comment]
:+1:

**Pull request merged**